File: model/funcionario.php
<?php

include_once 'usuario.php';

class Funcionario extends Usuario
{
    private $cargo;

    /**
     * Get the value of cargo
     */
    public function getCargo()
    {
        return $this->cargo;
    }

    /**
     * Set the value of cargo
     *
     * @return  self
     */
    public function setCargo($cargo)
    {
        $this->cargo = $cargo;

        return $this;
    }
}

File: model/Funcionario.py
from model.Usuario import Usuario
from controle.ControleApp import db

class Funcionario(Usuario):
    __tablename__ = 'Funcionario'
    id = db.Column(db.Integer, db.ForeignKey(Usuario.id), db.Sequence('Funcionario_seq'), primary_key=True)
    cargo = db.Column(db.String(50), nullable=False)

    __mapper_args__ = {
        'polymorphic_identity': 'Funcionario'
    }

